packages/gatsby-dev-cli/src/watch.js

```diff
@@ -50,9 +50,9 @@ function watch(root, packages, { scanOnce, quiet }) {
       const watchEvents = [`change`, `add`]
       if (_.includes(watchEvents, event)) {
         const [packageName] = filePath
-          .split(`packages/`)
+          .split(/packages[/\\]/)
```

Comment:
could use `path.sep` here - went with approach used above - https://github.com/gatsbyjs/gatsby/blob/7129c8683d667b7b73e1cdfbb66d592b4677d915/packages/gatsby-dev-cli/src/watch.js#L34

Reply:
I don't have a strong preference either way, but I kinda like the idea of using path.sep. But if both work, this is totally fine with me!